site/content/docs/commands/deploy.en.md

```diff
@@ -82,3 +82,19 @@ Initializes and deploys a service named "backend" to a "prod" environment.
 ```console
 $ copilot deploy --init-wkld --deploy-env=false --env prod --name backend
 ```
+
+Deploys all local, initialized workloads in no particular order.
+```console
+$ copilot deploy --all --env prod --name backend
+```
+
+Deploys multiple workloads in a prescribed order (fe and worker, then be).
+```console
+$ copilot deploy -n fe/1 -n be/2 -n worker/1
```

Comment:
in this case, are `fe` and `worker` parallel, or just in no particular order but both before `be`?

Reply:
Nvm. Found the answer to my question in your other PR!

Reply:
Is there validation on the numbers passed in with flags?

Reply:
I think the "validation" is when we try to convert the string to int. If the user passes `fe/s` it'd fail during the conversion. Other than that I don't think we do any additional validation - is it needed though?